Tests/SwiftParserTest/DeclarationTests.swift

```diff
@@ -2550,4 +2550,154 @@ final class DeclarationTests: ParserTestCase {
       """
     )
   }
+
+  func testDeclarationsWithPlaceholders() {
+    assertParse(
+      """
+      func 1️⃣<#name#>(2️⃣<#parameters#>) -> 3️⃣<#return type#> {
+          4️⃣<#function body#>
+      }
+      """,
+      substructure: FunctionDeclSyntax(
+        funcKeyword: .keyword(.func),
+        name: .identifier("name"),
+        signature: FunctionSignatureSyntax(
+          parameterClause: FunctionParameterClauseSyntax(
+            parameters: FunctionParameterListSyntax([])
+          ),
+          returnClause: ReturnClauseSyntax(
+            type: IdentifierTypeSyntax(
+              name: .identifier("return type")
```

Comment:
Does this test pass? The content of the identifier should be `<#return type#>`. And we’re missing `<#parameters#>` in the substructure.